xbmc/cores/RetroPlayer/rendering/VideoRenderers/RPBaseRenderer.cpp

```diff
@@ -129,8 +134,10 @@ void CRPBaseRenderer::ManageRenderArea(const IRenderBuffer& renderBuffer)
   const unsigned int sourceRotationDegCCW = renderBuffer.GetRotation();
   const float sourceAspectRatio =
       static_cast<float>(sourceWidth) / static_cast<float>(sourceHeight);
-
+  const SCALINGMETHOD scaleMOde = m_renderSettings.VideoSettings().GetScalingMethod();
```

Comment:
`scaleMode` vs `scaleMOde`, same below where it's used

Reply:
Yes, it was typo. I will fix this.